src/plugins/api/index.ts

```diff
@@ -1454,33 +1454,47 @@ export class MusicAssistantApi {
   }
 
   private handleResultMessage(msg: SuccessResultMessage | ErrorResultMessage) {
-    // Handle result of a command
     const resultPromise = this.commands.get(msg.message_id as number);
 
+    const isAuthAborted =
+      "error_code" in msg &&
+      typeof msg.details === "string" &&
+      msg.details === "auth_aborted";
+
+    const isAuthTimeout =
+      "error_code" in msg &&
+      typeof msg.details === "string" &&
+      msg.details === "auth_timeout";
+
```

Comment:
I don't think this may/should live here as its an implementation detail of the auth flow within config.
Also it will at some point cause confusion if our api is going to have auth on its own.
So I would move handling the auth error in the config view and maybe create a dedicated component which we can use for AddProvider and EditProviderConfig.